cluster-commands.go

```diff
@@ -151,7 +151,18 @@ type PeerInfo struct {
 	Name     string `json:"name"`
 	// Deployment ID is useful as it is immutable - though endpoint may
 	// change.
-	DeploymentID string `json:"deploymentID"`
+	DeploymentID string     `json:"deploymentID"`
+	SyncState    SyncStatus `json:"sync"` // whether to enable| disable synchronous replication
```

Comment:
why is this not a boolean? @poornas

Reply:
IMO we can make `enable/disable` an `mc` level construct instead of having this via SDK. It is very simple to just have `SyncReplication bool`

Reply:
Passing a bool would also entail an additional flag to know if isSyncReplicationSet - otherwise an endpoint change can overwrite the sync mode setting on the server

Reply:
Understood.